### PR TITLE
Community Page

### LINKED ISSUE #3049: Update Community Page
See [Figma](https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj/Website?node-id=7895%3A13381&t=QSjJjcVWe2SbLwzt-4)

### PR DESCRIPTION
**Pull request by Cahllagerfeld**
closes #3049 

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3042"></a>

### CODE REVIEW

File: src/lib/components/community/contribution-highlights.svelte [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  let selectedMonth = contributionMonths[contributionMonths.length - 1];
+  $: highlights = contributionHighlights.filter(
+    (highlight) => highlight.date === selectedMonth
+  );
+</script>
+
+<Section>
+  <div class="mb-small md:mb-medium">
+    <h2 class="text-center h2 mb-macro md:!mb-micro">
+      Contribution of the month
```
Comment (chrifro) [inline src/lib/components/community/contribution-highlights.svelte:19]: Should be "contributionS"

**Review by chrifro — COMMENTED** [review]
Nice ✨ thanks @Cahllagerfeld ! 

Several minor comments:

- [ ] can you reduce the top spacing between the navigation bar and h1? What about 72px instead of 160px? Currently the image is mostly cut on a 13" screen
Bildschirmfoto 2022-11-23 um 11 47 06

- [ ] The heroes image should not be cut on the sides. Ideally: it's cut on smaller viewports and fully visible on larger sizes
- [ ] `Join now ` CTA should link to gitpod.io/chat
- [ ] I don't think we should use the first quote "We had some issues with the wifi and @gitpod.." cc @pawlean 
- [ ] quote: ".@gitpod is 🔥" seems a bit weak and out of context. Let's remove that as well?  The other quotes are great!
- [ ] add hover effect to `September 2022` under contribution of the month
- [ ] I updated this image for the community heroes and included a bg (see mockup)
Bildschirmfoto 2022-11-23 um 11 57 55

**Mobile**
- [ ] reduce the spacing between CTA and image
Bildschirmfoto 2022-11-23 um 11 59 46

- [ ] I'd suggest that we remove some line breaks in this quote so that it is not that much bigger in height than the other quotes
Bildschirmfoto 2022-11-23 um 12 00 40

- [ ] the padding to the sides in this card seems too little compared to the other card styles
Bildschirmfoto 2022-11-23 um 12 02 31

- [ ] reduce the top and bottom padding inside the cards. For mobile, we should watch out to have as little spacing as possible to avoid any extra scrolling
Bildschirmfoto 2022-11-23 um 12 07 23

- [ ] dark theme: add a glow behind this image so that is has more contrast to the background
Bildschirmfoto 2022-11-23 um 12 10 47

- [ ] dark theme: the heroes image quality isn't the best and that's even more visible with the glow. I'll see if I can export it in a higher resolution. Otherwise I'd suggest removing the glow

**Comment by Cahllagerfeld** [comment]
Hey @chrifro 

>  I updated this image for the community heroes and included a bg (see mockup)

could you please remove the border-radius from the image in figma

> dark theme: add a glow behind this image so that is has more contrast to the background

I think I'm not able to add the glow, it would be awesome, if you could provide it.

- Regarding the two tweets, do we need to wait for input from @pawlean, or can we go ahead and delete them?

**Comment by chrifro** [comment]
I added the missing files to the Figma mockup [[1](https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj/Website?node-id=8158%3A14131&t=8uTtB6ux8oqJnhLI-1)]

>Regarding the two tweets, do we need to wait for input from @pawlean, or can we go ahead and delete them?

Let's delete them and open a follow-up PR in case Pauline disagrees and wants them back :)

**Comment by Cahllagerfeld** [comment]
I updated the PR, could you please recheck @chrifro

**Review by chrifro — APPROVED** [review]
I updated the heroes image in the Figma [component library](https://www.figma.com/file/VL0CJpHqcDJVJ4BEkoyRFV/Gitpod-Website-Component-Library?node-id=514%3A2677&t=SpQM6fNomHYjTMhi-1). The resolution should now be a lot better. Could you update the header image and use the one with the glow for dark theme? 
Sorry for the delay in the image update.

I left a few minor comments inlined. Otherwise, it's ready to be merged from my side 🙌

**Pull request merged**